src/GoogleAuthenticator.php

```diff
@@ -86,6 +86,21 @@ public function checkCode($secret, $code): bool
         return $result > 0;
     }
 
+    final protected function setPeriod($period):bool
+    {
+        if (!(null === $period || intval($period) < 1 || intval($period) > 86400)) {
+            $this->codePeriod = $period;
+            return true;
+        }
+
+        return false;
+    }
+
+    final protected function getPeriod():int
```

Comment:
Why protected?

Reply:
See the original issue.  Unless I'm reading it wrongly?